themes/2015/layouts/partials/navbar.html

```diff
@@ -1,6 +1,6 @@
 <div class="nav-brand">
     <a href="https://www.elao.com/" class="link-elao">
-        <img src="/images/arrow.svg"/>
+        <img src="/images/arrow.svg" alt="" />
```

Comment:
il y a déjà un libellé "agence", l'image arrow n'a pas de signification particulière, donc alt vide
"back to the elao website" serait éventuellement en title de la balise `<a>`